R/code_agreements.R

```diff
@@ -9,17 +9,15 @@
 #' @param date date column variable
 #' Ideally, date variable should come from a qPackage database/dataset
 #' for which dates were standardised with `standardise_dates()`
-#' @param dataset name of the dataset, optional. When provided with
-#' dataset argument, the function will return qID as a column in the dataset.
-#' If not provided, the function will return a character vector with qIDs.
+#' @return a character vector with the qIDs
 #' @importFrom usethis ui_done
 #' @importFrom stringr str_replace_all str_detect
 #' @importFrom purrr map
 #' @examples
 #' IEADB <- dplyr::slice_sample(qEnviron::agreements$IEADB, n = 10)
-#' code_agreements(IEADB$Title, IEADB$Signature)
+#' IEADB$qID <- code_agreements(IEADB$Title, IEADB$Signature)
 #' @export
-code_agreements <- function(title, date, dataset = NULL) {
+code_agreements <- function(title, date) {
```

Comment:
I really like the use of usethis::ui_done to inform the contributor what's happening under the hood. There seems to be some stylistic things to improve on here, but this can also wait. Then, lastly, it may be interesting to reoffer a `dataset` argument so that the function can just take an otherwise conforming qData dataset and pick out the title and date columns automagically.

Reply:
The dataset argument was removed because changing an environment item with a function was not that appropriate or straightforward. As for re-adding the dataset, we can do so that dataset find the title and a date column if none are specified. I will address these changes now.